inference-engine/tests/ie_test_utils/common_test_utils/ngraph_test_utils.cpp

```diff
@@ -83,6 +85,520 @@ std::string name(const Node& n) {
     return n->get_friendly_name();
 }
 
+namespace attr_comparison {
+
+using AttrName = std::string;
+
+class Result {
```

Comment:
nitpick: maybe ComparisonResult?

Reply:
It's `attr_comparison::Result`. Is this OK?